get-started/mcp/integrations.md

```diff
@@ -50,17 +50,29 @@ Before integrating with any client:
     5. Restart VS Code.
 
 === "Cursor"
-
-    1. Open Cursor settings and click on **MCP**.
-    2. Add server configuration using the following command:
-        
-        ```bash
-        npx mcp-remote https://api.kluster.ai/v1/mcp \
-        --header "Authorization: Bearer YOUR_MCP_TOKEN"
+    ![](/images/get-started/mcp/integrations/integrations-2.webp){ style="width:80%;" }
+
+    Open Cursor settings and:
+    
+    1. Select **Tools & Integrations**.
+
+    2. Choose **New MCP Server** and add the following configuration:
```

Comment:
```suggestion
    2. To add your first MCP, click **Add Custom MCP**. To add additional MCPs later, use **New MCP Server**. Then enter the following configuration:
```

The first time it just shows something a little bit different, so this covers all bases